lms/envs/aws.py

```diff
@@ -123,6 +123,11 @@
 
 CELERYBEAT_SCHEDULE = {}  # For scheduling tasks, entries can be added to this dict
 
+############################# Persistent Grades ####################################
+
+# Queue to use for updating persistent grades
+RECALCULATE_GRADES_ROUTING_KEY = LOW_PRIORITY_QUEUE
```

Comment:
Sorry - this needs to be ENV_TOKENS.get('RECALCULATE_GRADES_ROUTING_KEY',RECALCULATE_GRADES_ROUTING_KEY) which says pull from lms.env.json with a default of what you had already set in common.py.  Apologies if the syntax is a little wrong, typing on phone